src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -8924,6 +8924,28 @@ namespace Google.Example.Library.V1
         {
         }
 
+        /// <summary>
+        /// Test server streaming, non-paged responses.
+        /// gRPC streaming methods don't have an HTTP equivalent and don't need to have the google.api.http option.
+        /// </summary>
+        /// <param name="name">
+        /// The name of the shelf whose books we'd like to list.
+        /// </param>
+        /// <param name="callSettings">
+        /// If not null, applies overrides to this RPC call.
+        /// </param>
+        /// <returns>
+        /// The server stream.
+        /// </returns>
+        public virtual Book StreamBooks(
```

Comment:
Book seems to be an odd return type for a stream-based method. Is this really the intended behavior? Not some kind of stream?

Reply:
Ah drat, thanks for noticing this, it is indeed wrong.
I'll fix it now.